examples/player/simid_player.js

```diff
@@ -55,6 +56,13 @@ class SimidPlayer {
      */
     this.adComplete_ = adComplete;
 
+    /**
```

Comment:
minor nits. include a period after `is`
The tag is `@const` not constant
the const, private, and type can all go in the same line (`@const @private {boolean}`)